lib/linguist/languages.yml

```diff
@@ -1904,7 +1904,9 @@ Hack:
   codemirror_mode: php
   codemirror_mime_type: application/x-httpd-php
   extensions:
+  - ".hack"
```

Comment:
Is `.hack` considered the primary extension for Hack now?

Reply:
yes